pom.xml

```diff
@@ -16,7 +16,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.surefire.version>3.1.0</maven.surefire.version>
+        <maven.surefire.version>3.3.1</maven.surefire.version>
```

Comment:
Logically, there should be no need to upgrade Surefire at this point since all ABI will be identical with the next release.